frontend/src/lib/components/NavPanelAdvertisement/NavPanelAdvertisement.tsx

```diff
@@ -0,0 +1,136 @@
+                            we have, so we're gently bringing it to your attention :) Feel free to 
+                            dismiss it to make it go away."
+            >
+                <p className="text-xxs text-muted mb-1 p-0 text-right">ads via PostHog</p>
+            </Tooltip>
+            <Link
+                to={payload.app_link}
+                className="text-primary"
+                onClick={() => {
+                    posthog.capture('nav panel advertisement clicked', {
+                        product_key: payload.product_key,
+                        payload,
+                    })
+                }}
+            >
+                <div className="border rounded bg-primary text-xs *:flex *:gap-2 *:px-2 *:py-1">
+                    <div className=" justify-between mt-1">
+                        <div className="flex items-center gap-2">
+                            {product && getProductIcon(product.iconColor, product.icon, 'text-lg')}
+                            <strong>{payload.header || product?.name}</strong>
+                        </div>
+                        <LemonButton
+                            icon={<IconX className="text-muted" />}
+                            tooltip="Dismiss"
+                            tooltipPlacement="right"
+                            size="xxsmall"
+                            onClick={(e) => {
+                                e.preventDefault()
+                                e.stopPropagation()
+                                posthog.capture('nav panel advertisement dismissed', {
+                                    product_key: payload.product_key,
+                                    payload,
+                                })
+                                hideAdvertisement()
+                            }}
+                            noPadding
+                        />
+                    </div>
+
+                    <div className="flex flex-col gap-1">
+                        <p className="mb-0">{payload.text}</p>
+                        <p className="mb-0">
+                            <Link
+                                onClick={(e) => {
+                                    e.preventDefault()
+                                    posthog.capture('nav panel advertisement learn more clicked', {
+                                        product_key: payload.product_key,
+                                        payload,
+                                    })
+                                    window.open(payload.product_info_link, '_blank')
+                                }}
+                            >
+                                Learn more
+                            </Link>{' '}
+                            &middot;{' '}
+                            <Link
+                                to={payload.docs_link}
+                                onClick={(e) => {
+                                    e.preventDefault()
```

Comment:
Doesn't calling `preventDefault` here prevent you from navigation to `payload.docs_link`?